packages/story-editor/src/elements/video/display.js

```diff
@@ -118,19 +119,7 @@ function VideoDisplay({ previewMode, box: { width, height }, element }) {
           data-leaf-element="true"
         >
           {url && <source src={url} type={resource.mimeType} />}
-          {tracks &&
-            tracks.map(({ srclang, label, track: src, id: key }, i) => (
-              <track
-                srcLang={srclang}
-                label={label}
-                // Hides the track from the user.
-                // Displaying happens in MediaCaptionsLayer instead.
-                kind="metadata"
-                src={src}
-                key={key}
-                default={i === 0}
-              />
-            ))}
+          <Captions tracks={tracks} defaultKind="metadata" />
```

Comment:
This completely changes the semantics.

Before, we _force_ the `kind` to be `metadata`. Now this just says "use metadata as the default if none is provided"